.evergreen.yml

```diff
@@ -77,9 +91,40 @@ tasks:
       - func: "gen-notices"
       - func: "fetch goreleaser"
       - func: "package"
+      - command: shell.exec
+        params:
+          working_dir: mongocli/dist
+          script: |
+            set -o errexit
+            set -o verbose
+
+            # we could generate a similar name with goreleaser but we want to keep the vars evg compatibale to use later
+            mv mongocli_next_linux_i386.deb mongocli_next_${revision}_${version_id}_${created_at}_linux_i386.deb
+            mv mongocli_next_linux_i386.rpm mongocli_next_${revision}_${version_id}_${created_at}_linux_i386.rpm
+            mv mongocli_next_linux_i386.tar.gz mongocli_next_${revision}_${version_id}_${created_at}_linux_i386.tar.gz
+            mv mongocli_next_linux_x86_64.deb mongocli_next_${revision}_${version_id}_${created_at}_linux_x86_64.deb
+            mv mongocli_next_linux_x86_64.rpm mongocli_next_${revision}_${version_id}_${created_at}_linux_x86_64.rpm
+            mv mongocli_next_linux_x86_64.tar.gz mongocli_next_${revision}_${version_id}_${created_at}_linux_x86_64.tar.gz
+            mv mongocli_next_macOS_x86_64.tar.gz mongocli_next_${revision}_${version_id}_${created_at}_macOS_x86_64.tar.gz
```

Comment:
nit: maybe make `macos` all lowercase, just like all the other output files

Reply:
will do on a separate PR